source/module_io/read_wfc_lcao.h

```diff
@@ -0,0 +1,144 @@
+/**
+ * @file read_wfc_lcao.h
+ * @author kirk0830
+ * @brief This file belongs to the project of removal of class Local_Orbital_Wfc (LOWF)
+ * @version 0.1
+ * @date 2024-06-04
+ * 
+ * @copyright Copyright (c) 2024
```

Comment:
It seems that the copyright belongs to kirk0830, but for open-source code in fact it's complicated. I would suggest updating the description.

Reply:
Sorry it was copilot automatically added, I have deleted it in latest commit.